### DISCUSSION TITLE
When should be the new release with redis-py backend be published ?

### DISCUSSION THREAD

**Discussion by jGundermann** [abersheeran/asgi-ratelimit#58] [Q&A] [answered]
Hi! 

First of all thanks for this great package.
I'm currently working on a project that use python 3.10 and i have some trouble with the redis backend on 0.7.0.

As workaround i copied the new redis-py backend and test it on my project and it works like a charm. 
I wanted to know when you plan to release it?

have a nice day.
Kindly, 
Joris

**Comment by abersheeran** [comment, marked as answer]
😀Now version 0.8 has been released.

**Reply by jGundermann** [reply]
Thx :rocket:
